test/Makefile

```diff
@@ -24,3 +27,10 @@ $(TEST_PATH)/libxmp-test: $(T_OBJS)
 	if [ "$(V)" -gt 0 ]; then echo $$CMD; else echo LD $@ ; fi; \
 	eval $$CMD
 
+check-lite: $(TEST_PATH)/libxmp-lite-test
```

Comment:
It might be worth adding this to the distribution packages CI job as part of this patch, to verify it passes under CI. (It doesn't look like the MSVC/Watcom regression tests are currently building lite...)

Reply:
I pushed a commit adding this just to see how it turns out—it seemed fine locally.